docs/admin/templates/extending-templates/provider-authentication.md

```diff
@@ -42,6 +42,17 @@ environments:
 - [Amazon Web Services](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)
 - [Microsoft Azure](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs)
 - [Kubernetes](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs)
+- [Docker](https://registry.terraform.io/providers/kreuzwerker/docker/latest/docs)
+
+## Using remote Docker host
+
+You can use a remote Docker host in 2 ways.
+
+1. Configuring docker provider to use a
+   [remote host](https://registry.terraform.io/providers/kreuzwerker/docker/latest/docs#remote-hosts)
+   over SSH or TCP.
+2. Running an [external provisioner](../../provisioners.md) on the remote docker
+   host.
```

Comment:
coming back with some edit suggestions, but having a bit of trouble with my workspace - this is a placeholder

Reply:
I am also concerned if this the right doc to add these instructions. Feel free to suggest a better place to add them.

Reply:
I think it's a good place - we can add this same info to other spots as we find them, and since it's just reference (and not steps), there's little risk of duplication